tools/stonith_admin.c

```diff
@@ -375,266 +482,259 @@ validate(stonith_t *st, const char *agent, const char *id,
     return rc;
 }
 
+static pcmk__supported_format_t formats[] = {
+    PCMK__SUPPORTED_FORMAT_TEXT,
+    PCMK__SUPPORTED_FORMAT_XML,
+    { NULL, NULL, NULL }
+};
+
+static GOptionContext *
+build_arg_context() {
+    GOptionContext *context = NULL;
+    GOptionGroup *defn_group, *query_group, *fence_group, *addl_group;
+
+    pcmk__common_args_t args = { " - access the Pacemaker fencing API",
+                                 &options.version, &options.quiet, bump_verbosity,
+                                 "text (default), xml", &output_ty, &output_dest };
```

Comment:
I was thinking the struct would contain the storage, not just pointers, so we wouldn't need options.version, output_ty, etc.